book/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+import json
+from datetime         import timedelta, date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models          import Book
+
+
+class RecentlyBookView(View):
+    def get (self, request):
+        day    = request.GET.get('day')
+        limit  = request.GET.get('limit', '20')
+
+        today          = date.today()
+        previous_days  = today - timedelta(days=int(day))
+
+        books = Book.objects.filter(publication_date__range=
+                                    [previous_days,today]).order_by('-publication_date')[:int(limit)]
+
+        feedback = {
+            "recentlyBook" : [{
+                "title"  : book.title,
+                "image"  : book.image_url,
+                "author" : book.author
+            } for book in books]}
+        return JsonResponse(feedback, status=200)
```

Comment:
```suggestion
        books = [
            {
                "title"  : book.title,
                "image"  : book.image_url,
                "author" : book.author
            } for book in (Book.objects
                .filter(publication_date__range=[previous_days,today])
                .order_by('-publication_date')[:int(limit)])
        ]
        return JsonResponse("oneMonthBook" : books, status=200)
```

구조를 다시 한 번 살펴본 바, 굳이 사용하지 않아도 되는 변수 `feedback` 가 사용되었습니다.
첫째로 책 리스트를 표출하는 엔드포인트에 `feedback`이라는 변수는 결과를 표출하는 용도로 사용되기에는 직관적이지 못한 변수명인 것 같습니다.
두번째로, 변수를 하나 선언할 때마다 메모리를 사용하게 되는데요, 불필요한 변수를 선언하는 것은 불필요한 메모리를 사용하게 된다는 뜻이 됩니다 :) 따라서 꼭 필요하지 않은 변수를 지정하지 않고, 변수의 개수를 줄여나가는 연습을 지금부터 해보시는 것을 추천드립니다.

Reply:
@soheon-lee 
수정했습니다. 감사합니다.